src/citrine/informatics/predictors/auto_ml_predictor.py

```diff
@@ -0,0 +1,73 @@
+from typing import List, Optional
+
+from citrine._serialization import properties as _properties
+from citrine._serialization.serializable import Serializable
+from citrine._session import Session
+from citrine.informatics.data_sources import DataSource
+from citrine.informatics.descriptors import Descriptor
+from citrine.informatics.reports import Report
+from citrine.informatics.predictors import Predictor
+
+__all__ = ['AutoMLPredictor']
+
+
+class AutoMLPredictor(Serializable['AutoMLPredictor'], Predictor):
+    """A predictor interface that builds a simgle model.
+
+    The model uses the set of inputs to predict the responses.
+    Only one value for responses is currently supported.
+    Supported complex inputs (such as chemical formulas) are auto-featurized.
```

Comment:
This isn't true. Users will need to add a formula featurizer if they want to featurize chemical formulas.

Reply:
I copy pasta-ed this from the simple ml predictor - is it true for that one, or should I also remove it there?

Reply:
It is true SimpleML, but not for this one.

Reply:
SimpleML does a little extra work to connect the graph and featurize common descriptors that AutoML does not.